dev-libs/bglibs/files/bglibs-2.04-fix-feature-tests.patch

```diff
@@ -0,0 +1,172 @@
+diff -ru bglibs-2.04.orig/sys/trysigprocmask.c bglibs-2.04/sys/trysigprocmask.c
+--- bglibs-2.04.orig/sys/trysigprocmask.c	2024-04-06 07:26:44.552478736 +0000
++++ bglibs-2.04/sys/trysigprocmask.c	2024-04-06 07:27:39.755145808 +0000
+@@ -1,6 +1,6 @@
+ #include <signal.h>
+ 
+-main()
++int main()
```

Comment:
Should be `int main(void)` to fix the strict prototype warnings with Clang.

Reply:
Fixing. Also, Clang wasn't screaming at me about that?

Reply:
There used to be a USE flag to enable a stricter clang build, but it seems it was removed or otherwise changed? Using `-pedantic` in the `CFLAGS` will probably show it